Source/AliveLib/Bone.cpp

```diff
@@ -736,24 +736,16 @@ void BoneBag::vUpdate_412880()
 
     if (field_20_animation.field_92_current_frame == 2)
     {
-        if (field_120)
-        {
-            if (Math_NextRandom() < 40 || field_122)
-            {
-                field_120 = 0;
-                field_122 = 0;
-                SFX_Play_46FBA0(29u, 24, Math_RandomRange_496AB0(-2400, -2200));
-            }
-        }
-    }
-    else
-    {
-        field_120 = 1;
+      if (Math_NextRandom() < 40 || field_122_force_play_sound)
+      {
+        field_122_force_play_sound = 0;

```

Comment:
This seems to be changing behaviour as 

  field_120 = 0;	   

Is no longer set to 0?

Reply:
All references to `field_120` have been removed since the variable is redundant and not needed. From what I've tested, there seems to be no change in behaviour.